.github/workflows/conformance-aws-cni-v1.10.yaml

```diff
@@ -66,7 +66,7 @@ env:
   cilium_base_version: "1.10"
   cilium_cli_version: v0.10.7
   check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
-  eksctl_version: v0.94.0
+  eksctl_version: v0.101.0
```

Comment:
Do we also need to update the docs?

Reply:
We currently don't seem to suggest a particular `eksctl` version in the docs. Not sure whether we should?

Reply:
> Do we also need to update the docs?

Addressed in #20329